src/storage-types/src/errors.rs

```diff
@@ -400,11 +400,17 @@ impl Display for SourceErrorDetails {
             SourceErrorDetails::Initialization(e) => {
                 write!(
                     f,
-                    "failed during initialization, must be dropped and recreated: {}",
+                    "failed during initialization, source must be dropped and recreated: {}",
```

Comment:
Just to double check my understanding, this message is fine because these kind of errors are never retracted, right? Also, beyond user confusion these error strings are persisted as-is and so if they were ever retracted with a different message it wouldn't match the originally produced message

Reply:
Yes that is the case --- these should never be retracted according to the docstring: https://github.com/MaterializeInc/materialize/blob/8bed425d3273bf7e6262ff754febad8bac07d211/src/storage-types/src/errors.rs#L335-L340

Reply:
Cool, thanks for checking!